eu.ad5001.LogarithmPlotter.json

```diff
@@ -0,0 +1,43 @@
+{
+    "app-id": "eu.ad5001.LogarithmPlotter",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15",
+    "sdk": "org.kde.Sdk",
+    "finish-args": [
+        "--share=ipc",
+        "--device=dri",
+        "--socket=x11",
+        "--socket=wayland",
+        "--talk-name=org.freedesktop.DBus.Proprieties",
+        "--talk-name=org.freedesktop.IBus",
+        "--talk-name=org.freedestkop.portal.*",
+        "--filesystem=home"
+    ],
+    "command": "logarithmplotter",
+    "rename-desktop-file": "logarithmplotter.desktop",
+    "rename-icon": "logplotter",
+    "rename-appdata-file": "eu.ad5001.LogarithmPlotter.metainfo.xml",
+    "cleanup": [ ],
+    "post-install": [
+        "install -D mimetypes /app/share/mime/packages/x-logarithm-plot.xml"
```

Comment:
I don't think this works.
1. there is no `mimetypes` at the top level directory of the source tree.
2. the mimetype file should be name `${FLATPAK_ID}.xml`

Reply:
Is there any way to use something like "rename-mime-file" for this?

Reply:
sadly no.

Reply:
I see, is it possible to just rename the mime file in the build command and flatpak-builder would detect it immediately ?

Reply:
you can use `mv` if you prefer (but if it's not installed, `install` is best), but the `post-install` should be located in the module.

Reply:
Fixed in latest commit.